src/systemctl/systemctl.c

```diff
@@ -4253,7 +4257,24 @@ static void print_status_info(
         else
                 printf("\n");
 
-        if (endswith(i->id, ".timer")) {
+        is_timer = endswith(i->id, ".timer");
+
+        if (!is_timer && !strv_isempty(i->triggered_by)) {
+                char **trigger;
+                bool found = false;
+
+                STRV_FOREACH(trigger, i->triggered_by) {
+                        if (endswith(*trigger, ".timer")) {
```

Comment:
I have to agree with @boucman: limiting this to timers seems artificial. Maybe just removing this `if` here would be enough?